ui/src/kapacitor/components/RuleHandlers.tsx

```diff
@@ -132,6 +134,33 @@ class RuleHandlers extends PureComponent<Props, State> {
       <div className="rule-section">
         <h3 className="rule-section--heading">Alert Handlers</h3>
         <div className="rule-section--body">
+          <div className="rule-section--row rule-section--row-first rule-section--border-bottom">
+            <p>Alert Options:</p>
+            <div className="form-control-static">
+              <input
+                name="noRecoveries"
+                id="noRecoveries"
+                type="checkbox"
+                defaultChecked={rule.alertNodes.noRecoveries}
+                onClick={this.handleNoRecoveries}
+              />
+              <label htmlFor="noRecoveries">
+                Don't send alert on condition recovery
+              </label>
+            </div>
+            <div className="form-control-static">
+              <input
+                name="stateChangesOnly"
+                id="stateChangesOnly"
+                type="checkbox"
+                defaultChecked={rule.alertNodes.stateChangesOnly}
+                onClick={this.handleStateChangesOnly}
+              />
+              <label htmlFor="stateChangesOnly">
+                Send alert only when condition state changes
+              </label>
+            </div>
+          </div>
```

Comment:
This change looks like this:

![image](https://user-images.githubusercontent.com/16321466/153740343-fba95eff-fffb-4e01-ada2-5b6ea094890c.png)

which might make users a bit confused because:
- the new options appear first so that one might think that it is the most important thing of this section ... it is not, the options must be the last 
- the options appear fused, there should be some space to separate them
- the options appear even when no rule are selected

It will look better with these disruptions fixed:

![image](https://user-images.githubusercontent.com/16321466/153740586-2c2cff95-dff1-4fab-86b5-f1f56747f606.png)

I propose to have the new options under `{mappedHandlersOnThisAlert.length ? ` :
```jsx
          {mappedHandlersOnThisAlert.length ? (
            <>
              <div className="rule-message--endpoints rule-section--border-bottom">
                <HandlerTabs
                  handlersOnThisAlert={mappedHandlersOnThisAlert}
                  selectedHandler={selectedHandlerWithText}
                  handleChooseHandler={this.handleChooseHandler}
                  handleRemoveHandler={this.handleRemoveHandler}
                />
                <HandlerOptions
                  selectedHandler={selectedHandler}
                  handleModifyHandler={this.handleModifyHandler}
                  updateDetails={ruleActions.updateDetails}
                  rule={rule}
                  onGoToConfig={onGoToConfig}
                  onTestHandler={onTestHandler}
                  validationError={validationError}
                />
              </div>
              <div className="rule-section--row rule-section--row-last">
                <div className="form-control-static">
                  <input
                    name="stateChangesOnly"
                    id="stateChangesOnly"
                    type="checkbox"
                    defaultChecked={rule.alertNodes.stateChangesOnly}
                    onClick={this.handleStateChangesOnly}
                  />
                  <label htmlFor="stateChangesOnly">
                    Send alert only when condition state changes
                  </label>
                </div>
                <div className="form-control-static" style={{marginLeft: 5}}>
                  <input
                    name="noRecoveries"
                    id="noRecoveries"
                    type="checkbox"
                    defaultChecked={rule.alertNodes.noRecoveries}
                    onClick={this.handleNoRecoveries}
                  />
                  <label htmlFor="noRecoveries">
                    Don't send alert on condition recovery
                  </label>
                </div>
              </div>
            </>
          ) : null}
```